models/models.go

```diff
@@ -254,3 +254,28 @@ func MaxBatchInsertSize(bean interface{}) int {
 func Count(bean interface{}) (int64, error) {
 	return x.Count(bean)
 }
+
+// IsTableNotEmpty returns true if table has at least one record
+func IsTableNotEmpty(tableName string) (bool, error) {
+	return x.Table(tableName).Exist()
+}
+
+// DeleteAllRecords will delete all the records of this table
+func DeleteAllRecords(tableName string) error {
+	_, err := x.Exec(fmt.Sprintf("DELETE FROM %s", tableName))
+	return err
+}
+
+// GetMaxID will return max id of the table
+func GetMaxID(tableName string) (maxID int64, err error) {
```

Comment:
Can this be changed to `GetLastRepoID()`. I think that the table name should not be the concern of other packages. Same issue with the other three new functions.

Reply:
Maybe `GetTableLastID` because it's not only used by repository table.

Reply:
What I mean is that this kind of detail are the concern of the `models` package. What about?:
```
func GetMaxID(bean interface{}) (maxID int64, err error) {
```

and then

```
max, err := GetMaxID(&Repository{})
```